flask_smorest/spec/__init__.py

```diff
@@ -340,16 +347,37 @@ def _register_pagination_header(self):
 openapi_cli = flask.cli.AppGroup("openapi", help="OpenAPI commands.")
 
 
+def _get_api():
+    return current_app.extensions["flask-smorest"]["ext_obj"]
+
```

Comment:
I wouldn't have bothered factoring this but this is merely a matter or personal preference.

Reply:
From my point of view when something (even a single line) is copy/pasted more then two times, it makes sense to extract it somehow.

Reply:
In original co

> Thanks for this.
> 
> Do you think it could be possible to have a single print command with format as an optional argument defaulting to JSON (and likewise for write)?
> 
> This would make for an (arguably) nicer user interface. It might also help factorize code, although this is not my concern, here.

I think it's a good idea! I didn't think about it. I will update the PR today or tomorrow.

Do you think it would be possible to release soon after the PR is merged, so I can use these commands?

Reply:
Can't make promises but I'll try to ship fast.

Regarding the duplicated line, it is just a dict access and it could be argued that the function call hinders readability a bit. But no strong objection. (In any case, I can always rework it someday if I really feel the need.)